webpacker_guide.md

```diff
@@ -14,16 +14,75 @@ After reading this guide, you will know:
 
 ## What Is Webpacker?
 
+Webpacker is a Rails wrapper around the [webpack](https://webpack.js.org) build system that provides a standard webpack configuration and reasonable defaults. 
 
+### What is webpack?
+
+The goal of webpack, or any front-end build system, is to allow you to write your front-end code in a way that is convenient for developers and then package that code in a way that is convenient for browsers. With webpack, you can manage JavaScript, CSS, and static assets like files or fonts. Webpack will allow you to write your code, reference other code in your application, transform you code, and combine your code into easily downloadable packs.
 
 ## How is Webpacker Different from Sprockets?
 
+Rails also ships with Sprockets, an asset-packaging tool whose features overlap with Webpacker. Both tools will compile your JavaScript into into browser-friendly files, and minify and fingerprint them in production. Both tools allow you to incrementally change files in development.
```

Comment:
Might be worth mentioning here that sprockets is here to stay (for the immediate future?) for CSS and images. There's a lot written up by DHH and others on the WTF forum about the reasoning for that.